flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -513,6 +501,15 @@
                     <POTENTIOMETER>88</POTENTIOMETER>
                 </UseTemplate>
 
+                <UseTemplate Name="ASOBO_GT_Component_Emissive_Gauge">
+                    <NODE_ID>SCREEN_PFD_L</NODE_ID>
+                    <POTENTIOMETER>88</POTENTIOMETER>
+                    <PART_ID>SCREEN_PFD1</PART_ID>
+                    <CAMERA_TITLE>PFD</CAMERA_TITLE>
+                    <!-- Pixelated screens needs more light -->
+                    <EMISSIVE_CODE>2</EMISSIVE_CODE>
```

Comment:
Doesn't `EMISSIVE_CODE` break the potentiometers ?

Reply:
No. Potentiometers works fine. This value acts as a "`SET_CODE`", sort of, and this value gets multiplied on top of the value that controls the emissive. If you walk up the template tree you see where I end up.

Reply:
@Benjozork Just for reference, in addition to the above:
```
<UseTemplate Name="ASOBO_GT_Emissive_Gauge">
    <EMISSIVE_CODE>(A:LIGHT POTENTIOMETER:#POTENTIOMETER#, Percent over 100) #EMISSIVE_CODE# *</EMISSIVE_CODE>
</UseTemplate>
```
This is where that snippet ends up in the end.